recipes/pyrad_mch/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyrad_mch" %}
+{% set version = "0.4.4" %}
+{% set sha256 = "c806405acf1d5ea6f18b6e6363123ea6a3d88070fa39759528108fa594e2fa70" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k or win]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+  host:
+    - python
+    - pip
+    - pyart_mch
```

Comment:
Is this needed to install?

Reply:
Pyrad is basically a wrapper to pyart.

Reply:
sure, but do you need it to build the actual package, or is it only a runtime dependency?